api/billingd/service/service.go

```diff
@@ -362,15 +376,16 @@ func (s *Service) Stop(force bool) error {
 }
 
 type Customer struct {
-	Key        string `bson:"_id"`
-	CustomerID string `bson:"customer_id"`
-	ParentKey  string `bson:"parent_key"`
-	Email      string `bson:"email"`
-	Status     string `bson:"status"`
-	Balance    int64  `bson:"balance"`
-	Billable   bool   `bson:"billable"`
-	Delinquent bool   `bson:"delinquent"`
-	CreatedAt  int64  `bson:"created_at"`
+	Key         string `bson:"_id"`
+	CustomerID  string `bson:"customer_id"`
+	ParentKey   string `bson:"parent_key"`
+	Email       string `bson:"email"`
+	AccountType string `bson:"account_type"`
```

Comment:
How about using the [account type](https://github.com/textileio/textile/blob/master/mongodb/accounts.go#L45) directly?

Reply:
what does that look like in the request object? just convert it to int64 and then back?

Reply:
I think so, yep. Your new client option could take `mdb.AccountType`, the just wrap that it `int64()` in the client method, the when creating the customer, `mdb.AccountType(req.AccountType)`

Reply:
k, done + rebased